common/traits/hfe_leader_trait.txt

```diff
@@ -19,7 +19,10 @@ hfe_ruler_trait_genomod = {
 		hfe_trigger_leaders_traits_enabled = yes
 		exists = from
 		from = {
-			is_gestalt = no
+			OR = {
+				is_gestalt = no
+				has_authority = auth_hive_mind
```

Comment:
This is ruler trait

Reply:
What does that have to do with not being a Gestalt Empire?

Reply:
Everything.
Hive minds have no ruler traits.

Reply:
Okay, done